keyvalue/clients/src/main/java/tachyon/client/keyvalue/KeyValueStoreWriter.java

```diff
@@ -48,4 +51,15 @@
    */
   void put(ByteBuffer key, ByteBuffer value) throws IOException, TachyonException;
 
+  /**
+   * Merges a key-value store to the current store and deletes the merged store afterwards.
+   *
+   * @param uri URI of the key-value store to be merged
+   * @throws IOException if non-Tachyon error occurs
+   * @throws FileDoesNotExistException if the uri does not exist
```

Comment:
we could combine FileDoesNotExistException and IsNotKeyValueStoreException for this method and call the result KeyValueStoreDoesNotExistException? The implementation could catch FileDoesNotExistException and rethrow as KeyValueStoreDoesNotExistException when it's looking for a key-value store